components/atoms/NegativeButton.tsx

```diff
@@ -24,6 +24,8 @@ export const NegativeButton = (props: Props): React.ReactElement => {
       borderWidth={1}
       borderColor={colors.primaryTeal}
       borderRadius={RADIUS_BUTTON}
+      backgroundColor={colors.white}
```

Comment:
Delete the following two lines of code. After that, you can set the `backgroundColor` and `color` as shown below:

 ```jsx
<NegativeButton backgroundColor="#fff"  color="#fff"/>
```

```suggestion
      backgroundColor={colors.white}
      color={colors.textButton}
```